### PR TITLE
mask the import names for less likelihood of conflict

### LINKED ISSUE #145: Variable inspector clears "tf" as a variable after cell run
At first I thought this was a tensorflow problem, but later realized I can't store anything into a variable named "tf" without having it cleared.  This seems to only happen with variable-inspector is enabled.  See the simple 2-cell notebook below.  Notice how somewhere between running those two cells, tf got overwritten with None
image

I'm using v0.5.0 of this plugin with jupyterlab 2.1.2

### PR DESCRIPTION
**Pull request by timkpaine**
fixes #145, #131, future similar issues

also fails over keras import to `tensorflow.keras` since its part of the TensorFlow package

### CODE REVIEW

**Comment by timkpaine** [comment]
Screen Shot 2020-05-29 at 10 24 18
